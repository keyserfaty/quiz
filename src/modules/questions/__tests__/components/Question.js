import React from 'react'
import Questions from '../../components/Question/index'

import renderer from 'react-test-renderer'

test('Should return a List element', () => {
  const tree = renderer.create(<Questions />).toJSON()
  expect(tree).toMatchSnapshot()
})