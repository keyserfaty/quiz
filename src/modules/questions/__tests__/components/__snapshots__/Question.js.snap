// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should return a List element 1`] = `
<View
  style={
    Object {
      "flex": 1,
    }
  }
>
  <Text
    style={Object {}}
  >
    Entertainment: Videogames
  </Text>
  <View
    style={Object {}}
  >
    <Text>
      Some question text
    </Text>
  </View>
  <Text
    style={Object {}}
  >
    1 / 10
  </Text>
  <View
    style={Object {}}
  >
    <View
      accessible={true}
      isTVSelectable={true}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignSelf": "center",
          "backgroundColor": "rgba(255, 255, 255, 1)",
          "borderRadius": 50,
          "flexDirection": "row",
          "justifyContent": "center",
          "opacity": 1,
          "paddingHorizontal": 30,
          "paddingVertical": 15,
          "width": 235,
        }
      }
    >
      <Text
        style={
          Array [
            Object {
              "fontFamily": "Poppins-Regular",
              "fontSize": 18,
              "letterSpacing": 2,
              "textAlign": "center",
            },
            Object {
              "color": "rgba(55,175,255,0.87)",
            },
            undefined,
          ]
        }
      >
        NO
      </Text>
    </View>
    <View
      accessible={true}
      isTVSelectable={true}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignSelf": "center",
          "backgroundColor": "rgba(255, 255, 255, 1)",
          "borderRadius": 50,
          "flexDirection": "row",
          "justifyContent": "center",
          "opacity": 1,
          "paddingHorizontal": 30,
          "paddingVertical": 15,
          "width": 235,
        }
      }
    >
      <Text
        style={
          Array [
            Object {
              "fontFamily": "Poppins-Regular",
              "fontSize": 18,
              "letterSpacing": 2,
              "textAlign": "center",
            },
            Object {
              "color": "rgba(55,175,255,0.87)",
            },
            undefined,
          ]
        }
      >
        YES
      </Text>
    </View>
  </View>
</View>
`;
